x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { useValues } from 'kea';
+import { EuiSpacer, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+// @ts-expect-error types are not available for this package yet;
+import { SearchProvider, SearchBox, Sorting } from '@elastic/react-search-ui';
+// @ts-expect-error types are not available for this package yet
+import AppSearchAPIConnector from '@elastic/search-ui-app-search-connector';
+
+import './search_experience.scss';
+
+import { EngineLogic } from '../../engine';
+import { externalUrl } from '../../../../shared/enterprise_search_url';
+
+import { SearchBoxView, SortingView } from './views';
+import { SearchExperienceContent } from './search_experience_content';
+
+const DEFAULT_SORT_OPTIONS = [
+  {
+    name: 'Recently Uploaded (desc)',
```

Comment:
This needs to be i18n'ed - I also would suggest moving it to a `constants` file

Reply:
rats

Reply:
c440114a59df8fe473ef14dfee4e3b6d1ac2725f